internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/TestBatch.java

```diff
@@ -986,7 +987,23 @@ public void testLargeValues() throws Exception {
     // Verify that after records are chunked and re-assembled, the original sizes of these records are being recorded
     // to the metrics sensor, and are within the correct size range.
     int minSize = 1024 * 1024; // 1MB apiece
-    validatePerStoreMetricsRange(storeName, ASSEMBLED_RECORD_VALUE_SIZE_IN_BYTES, minSize, MAX_RECORD_VALUE_SIZE);
+    validatePerStoreMetricsRange(storeName, ASSEMBLED_RECORD_VALUE_SIZE_IN_BYTES, minSize, LARGE_RECORD_VALUE_SIZE);
+  }
+
+  /** Test that values that are too large will fail the push job only when the limit is enforced. */
+  @Test(dataProvider = "True-and-False", dataProviderClass = DataProviderUtils.class, timeOut = TEST_TIMEOUT)
+  public void testStoreWithTooLargeValues(boolean enforceLimit) throws Exception {
+    final int wayTooLargeRecordValueSize = 11 * 1024 * 1024; // 11 MB
+    final int maxRecordSizeBytesForTest = (enforceLimit) ? 10 * 1024 * 1024 : -1; // -1 means no limit
```

Comment:
Can we replace `1024 * 1024` with `ByteUtils.BYTES_PER_MB` for readability?

Reply:
Omg I've been looking for that for so long! Replaced.